addons/l10n_ec/data/account_chart_template_setup_accounts.xml

```diff
@@ -3,7 +3,7 @@
     <data noupdate="1">
 
         <record id="l10n_ec_ifrs" model="account.chart.template">
-            <field name="name">Plan Contable NIIF Ecuador conforme Superintendencia de Compañías</field>
+            <field name="name">Ecuador - Plan Cuentas NIIF SupCias</field>
```

Comment:
It is the same as before, but better lol. I changed it because in multicountry projects only Ecuador doesn't "start" with its country name (like Peru, Chile, Colombia), so the Ecuadorian package it will be easier to find in that kind of project, and shortened the name so it fits in the field like the other countries.

Reply:
ok, good to know